openfpga/src/repack/repack_option.cpp

```diff
@@ -46,6 +46,55 @@ bool RepackOption::is_pin_ignore_global_nets(const std::string& pb_type_name,
   return false;
 }
 
+bool RepackOption::net_is_specified_to_be_ignored(std::string cluster_net_name,
+                                                  std::string pb_type_name,
+                                                  const BasicPort& port) const {
+  auto pin_ctx_to_parse =
+    design_constraints_.ignore_net_on_pin(cluster_net_name);
+  if (pin_ctx_to_parse.empty()) {
+    /* Not found, return false */
+    return false;
+  } else {
+    /* Split the content using a tokenizer */
+    for (auto pin_ctx_to_parse_iter : pin_ctx_to_parse) {
+      int num_parse_errors_temp = 0;
+      StringToken tokenizer(pin_ctx_to_parse_iter);
+      std::vector<std::string> tokens = tokenizer.split(',');
```

Comment:
I believe the tokenizer for ',' and '.' can be handled in ``read_xml_repack_constraints.cpp`` or ``repack_design_constraints.cpp'. The syntax defined here is applicable and limited to parsers. Later, when we support a different parser or file format, different syntax may be applied.
When we load data into repack options,  the data should be already processed. We should limit the requirements on syntax in repack options.